ecommerce_integrations/shopify/page/shopify_import_products/shopify_import_products.js

```diff
@@ -138,7 +138,7 @@ shopify.ProductImporter = class {
 					focusable: false,
 				},
 				{
-					name: 'SKUs',
+					name: 'Variant SKUs',

```

Comment:
I think the current version is fine as it is? 

Not all products have variants and for them, single SKU will be shown.

Reply:
Okay got it. In Shopify products that have variants, have only variant SKUs and the parent item does not have a SKU whereas for single product it has a SKU. I think SKU as the column title is fine in that case. Closing this PR.